test/simpleTestForGenerator/BSD-3-Clause.txt

```diff
@@ -6,6 +6,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
 
-3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission. To obtain permission, contact <email>.
```

Comment:
Same comment as in XML file, I wouldn't recommend adding this to the test text file.